docker-compose.yml

```diff
@@ -17,5 +17,23 @@ services:
     networks: 
       - user-network 
 
+  matches:
+    build: ./matches
+    ports:
+      - "81:80" 
+    networks: 
+      - matches-network 
+    
+  matches-db:
+    image: postgres:12.1 
+    environment:
+      - PGUSER=postgres
+    volumes:
+      - ./matches-db/init.sql:/docker-entrypoint-initdb.d/init.sql
+    networks: 
+      - matches-network 
+  
+
 networks:
   user-network:
+  matches-network:
```

Comment:
Why are these empty keys rather than a list?

Reply:
That just defines the network (https://docs.docker.com/compose/networking/) - Don't think we need to do anything custom with them, so they're just empty for now